theano/compile/function.py

```diff
@@ -100,6 +100,9 @@ def function(inputs, outputs=None, mode=None, updates=None, givens=None,
         If True, do not perform any automatic update on Variables. If False
         (default), perform them all. Else, perform automatic updates on all
         Variables that are neither in "updates" nor in "no_default_updates".
+    accept_inplace : bool
+        True iff the graph can contain inplace operations prior to the
+        optimization phase (default is False).
```

Comment:
IMO, keep the description and add a warning that it is not recommended.
There are use cases where it can be useful, especially when cloning or unpickling graphs that are already optimized, but it is not the typical case.